packages/amplication-client/src/Layout/InnerTabLink.tsx

```diff
@@ -8,14 +8,20 @@ type Props = {
   children: React.ReactNode;
   icon: string;
   to: string;
+  onClick?: (selectedItem: any) => void;
```

Comment:
if we decide to use the OnClick event, I think we should use a data object to pass back to the parent. Otherwise, I don't think it will be usable to the parent 

have a look at this for example: 
packages/amplication-design-system/src/components/SelectMenu/SelectMenu.tsx

Reply:
@yuval-hazaz we don't need it